### ISSUE TITLE
How do you attach the pieces together ?

### ISSUE THREAD

**Issue by berserkwarwolf** [aliuliu121/NeonSFF#1]
You say you printed on a bed with a volume of 180mm x 180mm x 150mm, but how did you assemble the parts afterwards ? Did you glue them together ? If so, I'd like to help by adding a proper mounting system, using the same M3 screws and inserts.

**Comment by aliuliu121** [comment]
Yes, I glued them together using a combination of CA glue and heatwelding
the parts together using a soldering iron.

**Comment by berserkwarwolf** [comment]
How do you recommend I add the mounting system ? I'm thinking of something like the modivio case.
